specs/governance/alligator.md

```diff
@@ -0,0 +1,61 @@
+# Alligator
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+- [Subdelegations](#subdelegations)
+- [Token Transfer Hook](#token-transfer-hook)
+- [View functions](#view-functions)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+The `Alligator` contract implements subdelegations that can be used by tokens contracts that inherit
+[`GovernanceToken`](gov-token.md). Subdelegations allow for advanced delegation use cases, such as partial,
+time-constrained & block-based delegations, relative & fixed allowances, and custom rules. The `Alligator` contract
```

Comment:
Alligator only requires migrating delegation state from the token contract for backwards compatibility, this isn't clear here. Its important to call that out because fresh chains don't need that feature

Reply:
I think you should move all of the backwards compat stuff out of a the overview and into a "Backwards Compatibility" section

Reply:
all good ideas @tynes!